frontend/src/tabs/zoom.pug

```diff
@@ -1,10 +1,17 @@
 #zoom
-  h1 {{ info.user.displayName }} ({{ info.user.name }})
-  a(v-on:click="openSummary") Show ramp chart for this period
-
   .zoom__title
-    .zoom__title--repo {{ info.user.repoName }} &nbsp;
-    .zoom__title--granularity.mini-font (granularity: one ramp per {{ filterTimeFrame }})
+    .zoom_title--group
+      h1(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
+      h1(v-else) {{ info.user.repoName }}
```

Comment:
The font size for the header doesn't seem to be the same as that in the authorship view. I think it would be good if they both had the same font size to give a more uniform look.

Reply:
Yes, it would be better for the authorship and zoom tab to be the same, that way a common tab panel style sheet can be used.

Reply:
@jamessspanggg u have missed out the comment here. 
Not necessary to invent a common tab panel style sheet yet but do try to make the headers similar to each other.

Reply:
Sorry for missing out this. I have standardised the fonts used and created two global fonts `large-font` and `medium-font`.